google-translate-connector-product/README.md

```diff
@@ -0,0 +1,21 @@
+# google-translate-connector
+
+Say goodbye to language barriers in process automation with Axon Ivy's translation service powered by Google Translate. The connector seamlessly integrates into your existing workflows, allowing you to communicate easily regardless of the language. This marketplace item:
+
+- Supports you with an easy-to-copy demo implementation to reduce your integration effort.
+- Enables low-code citizen developers to provide multi-lingual user interfaces.
+## Demo
+
+Translates texts via API.
+
+## Setup
+
+Prepare a Google Translate Key API
```

Comment:
please give guidance where to get the google translate api key: do I need a google account? ... where do i have to sign up for an api key? ... if the integration isn't for free, you may give the user a hint on that early on here on the setup instructions.

Reply:
updated